themes/tdr/login/info.ftl

```diff
@@ -7,18 +7,16 @@
             ${message.summary}
         </#if>
     <#elseif section = "form">
-        <div id="kc-info-message" class="govuk-body">
-            <p class="instruction">${message.summary}<#if requiredActions??><#list requiredActions>: <b><#items as reqActionItem>${msg("requiredAction.${reqActionItem}")}<#sep>, </#items></b></#list><#else></#if></p>
-            <#if skipLink??>
-            <#else>
-                <#if pageRedirectUri?has_content>
-                    <p><a href="${pageRedirectUri}">${kcSanitize(msg("backToApplication"))?no_esc}</a></p>
-                <#elseif actionUri?has_content>
-                    <p><a href="${actionUri}">${kcSanitize(msg("proceedWithAction"))?no_esc}</a></p>
-                <#elseif (client.baseUrl)?has_content>
-                    <p><a href="${client.baseUrl}">${kcSanitize(msg("backToApplication"))?no_esc}</a></p>
-                </#if>
+        <p class="govuk-body">${message.summary}<#if requiredActions??>:</#if></p>
```

Comment:
Is this isolated colon still necessary? What does this look like in both cases?

Actually, it would be helpful to see some screenshots of all these pages if you have some. It'd make it easier to wrap my head around the change, given that some of the text comes from Keycloak.

Reply:
Screenshots attached to the PR comment